server/app/services/openApi/v2/Parameter.java

```diff
@@ -0,0 +1,89 @@
+package services.openApi.v2;
+
+import com.google.auto.value.AutoValue;
+import java.util.Optional;
+
+/**
+ * Describes a single operation parameter.
+ *
+ * <p>https://swagger.io/specification/v2/#parameter-object
+ */
+@AutoValue
+public abstract class Parameter {
+  /**
+   * Required.
+   *
+   * <p>The name of the parameter. Parameter names are case sensitive.
+   *
+   * <p>If in is "path", the name field MUST correspond to the associated path segment from the path
+   * field in the Paths Object. See Path Templating for further information.
```

Comment:
Can we link out somewhere, or add more context for where to look for more info on Path Templating?

Reply:
I pulled in the comments from the Swagger Spec. I thought it would be useful to have closer to the code and for code-completion in IDEs. If you go to the link at the class level it'll have the details and a link to the [path-templating](https://swagger.io/specification/v2/#path-templating). 

If you think it's important enough to add as a link here, I should probably do a full pass and make sure all the comments have links where applicable. What I'd do is create a sub-issue to update v2/v3 comments at the same time and as a separate commit I think.

Reply:
I agree that having the comments close to the code is better, and it's definitely ok to leave linking and formatting as a nice-to-have to do later.